Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -56,6 +56,10 @@
 #include "UICommon/GameFile.h"
 
 QPointer<MenuBar> MenuBar::s_menu_bar;
+const QString MenuBar::SIGNATURE_SELECTOR =
+    tr("Dolphin Signature File (*.dsy)") + QStringLiteral(";;") +
+    tr("Dolphin Signature CSV File (*.csv)") + QStringLiteral(";;") +
+    tr("WiiTools Signature MEGA File (*.mega)");
```

Comment:
This would run at the beginning of the program execution, right? `tr` doesn't work at that point because the translation handler hasn't been set up yet.

Reply:
I forgot about that, I replaced it with a private member function.